kubernetes/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.13 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
```

Comment:
`ADD` instead of `COPY`

Reply:
I think we are only copying local files to docker image, `COPY` can be used here because it’s more explicit. If there's any URL or extraction ops, we can change to `ADD`